apis/launchers.md

```diff
@@ -61,48 +61,80 @@ As you can see, the method `set_progress` takes a `double` value type and is a r
 
 ## Actions
 
-Actions are specific functions your app can perform without already being open; think of them as alternate and more specific entry points into your app. Actions appear in the context menu of your app icon in the Applications Menu and Dock, and are searchable by name from the Applications Menu.
+You can use any action defined in the `app` namespace as one of the entry points for your application. They appear in the context menu of your app icon in the Applications Menu and Dock, and are searchable by name from the Applications Menu. Implementing actions is covered in-depth in [the actions section](actions).
```

Comment:
I reverted the intro back to its original wording. I also pulled the cross-link to the Actions page down into the _Declaring actions_ sub-section, since that's the first time this info is actually useful.